exporters/tokens/src/helpers/colorHelper.ts

```diff
@@ -73,3 +75,21 @@ export const transformColorsToVariables = (
 
   return transformedValue;
 };
+
+export const filterColorCollections = (tokens: Token[]) => {
+  const COLLECTION_NAME = 'Theme tokens';
+
+  return tokens.filter((item) => {
+    // Include all tokens that are not of tokenType "Color"
+    if (item.tokenType !== TokenType.color) return true;
+
+    // Proceed with filtering for "Color" tokenType
+    const collectionProperty = item.properties.find((prop) => prop.name === 'Collection');
```

Comment:
**suggestion (if-minor):** `Collection` is a magic constant. I think we should use a constant instead of this and name it according to the value.

Reply:
moved to the constant